parity/configuration.rs

```diff
@@ -548,7 +548,7 @@ impl Configuration {
 			apis: {
 				let mut apis = self.args.flag_ipcapi.clone().unwrap_or(self.args.flag_ipc_apis.clone());
 				if self.args.flag_geth {
-					apis.push_str("personal");
+					apis.push_str(",personal");
```

Comment:
what if `apis` is empty? E.g. someone disabled all the APIs but still running with `--geth` because of other compatibility issues